source/_includes/installation/operating_system.md

```diff
@@ -12,8 +12,8 @@ Follow this guide if you want to get started with Home Assistant easily or if yo
 
 We will need a few things to get started with installing Home Assistant. Links below lead to Amazon US. If you’re not in the US, you should be able to find these items in web stores in your country.
 
-- [Raspberry Pi 4](https://amzn.to/2S0Gcl1) (Raspberry Pi 3 is ok too, if you have one laying around). Raspberry Pi are currently hard to come by, use [RPilocator](https://rpilocator.com/?cat=PI4) to find official distributors with stock.
-- [Power Supply for Raspberry Pi 4](https://amzn.to/2ReZ2Vq) or [Power Supply for Raspberry Pi 3](https://amzn.to/2R8yG7h)
+- [Raspberry Pi 4](https://amzn.to/2S0Gcl1). Raspberry Pi are currently hard to come by, use [RPilocator](https://rpilocator.com/?cat=PI4) to find official distributors with stock.
```

Comment:
Sorry, but going to friendly decline this change.

The Raspberry Pi3 is in active support and in active use by 14% of our user base (see also https://analytics.home-assistant.io for our public analytics on this).

It depends on your needs and what you want to do with them. (Even an RPi4 can be unsuitable if you like to do, for example, lots of cameras).

That combined with the massive chips shortage making it hard to get devices atm, making a Pi3 that is around still a valid and supported device to start an Home Automation journey.

../Frenck

Reply:
@frenck fair enough. Are there required minimum specs listed somewhere? It feels strange to be told that 500MB of RAM isn't enough to run HA, even though some Pi 3s have 500MB and overall Pi 3 is listed as supported. Maybe the answer is that Pi 3 As should be ruled out? Or perhaps the setup instructions need to be augmented with instructions do do SSD + larger swap file for Pi 3s (seems to be the recommended solution that keeps cropping up on all the posts I've seen where people have the same issue as me)?

Happy to make either of those changes if you think they're reasonable.

Reply:
> It feels strange to be told that 500MB of RAM isn't enough to run HA

It can be enough, depending on what you do with it. If you want to control your lights and such in your home (doing Home Automation as for what Home Assistant is all about), that will be enough. It is what you are doing more with it.

As lined out above as well, there is no universal rule for that.

../Frenck

Reply:
> It is what you are doing more with it.

Nope, I have one automation uses a philips hue remote button click to turn off some lights and call google home via the SDK. No addons installed. I think I've got a super minimal setup, but even that seems to be too much for a Pi 3 A+